RELEASE_NOTES.rst

```diff
@@ -8625,7 +8625,7 @@ Bug fixes
 - [AIRFLOW-2645][AIRFLOW-2617] Add worker_container_image_pull_policy
 - [AIRFLOW-2661] fix config dags_volume_subpath and logs_volume_subpath
 - [AIRFLOW-3550] Standardize GKE hook (#4364)
-- [AIRFLOW-2863] Fix GKEClusterHook catching wrong exception (#3711)
+- [AIRFLOW-2863] Fix GKEHook catching wrong exception (#3711)
```

Comment:
can you revert the unrelated change please

Reply:
I have removed these changes